test/src/util/test_pelz_uri_helpers.c

```diff
@@ -42,27 +43,37 @@ void test_scheme_extraction(void)
 
   uriParseSingleUriA(&uri, pelz_uri, NULL);
   CU_ASSERT(get_uri_scheme(uri) == PELZ_URI);
+  pelz_log(LOG_DEBUG, "Finish URI functions");
```

Comment:
Why are we logging in the tests rather than relying on the test outputs?

Reply:
This is logged so if a change causes the test to fault then you can better determine the fault location from the debug logs.